src/sidebar/util/oauth-client.js

```diff
@@ -151,10 +151,12 @@ class OAuthClient {
    *
    * Returns an authorization code which can be passed to `exchangeAuthCode`.
    *
-   * @param {Window} $window
+   * @param {Window} $window - Window which will receive the auth response.
+   * @param {Window} authWindow - Popup window where the login prompt will be shown.
+   *   This should be created using `createLoginPopupWindow`.
    * @return {Promise<string>}
    */
-  authorize($window) {
+  authorize($window, authWindow) {
```

Comment:
The variation between `auth` (`authWindow`) and `login` (`createLoginPopupWindow`) — curious about that: is there a difference between the two (i.e. would calling this param `loginWindow` be _wrong_?). This light confusion is compounded by the fact that it's `$window` that "receive[s] the auth response", ultimately.

Reply:
The URL shown in the popup is what the OAuth spec terms the "authorization endpoint". The client uses the term "Log in" in the UI for user friendliness.

Is it clearer if I rename `createLoginPopupWindow` to `openAuthPopupWindow`?

Reply:
I think that would feel more consistent to me, yep!